### ISSUE TITLE
Update alowed python versions for `onnx-graphsurgeon`

### ISSUE THREAD

**Issue by tadejsv** [NVIDIA/TensorRT#3781]
`onnx-graphsurgeon` only works with python versions up to 3.8. Please upgrade it to at least 3.10 - this is the Python version used in all latest Nvidia containers

**Comment by lix19937** [comment]
pip install nvidia-pyindex      
pip install onnx-graphsurgeon

**Comment by lix19937** [comment]
or   https://developer.nvidia.com/tensorrt/download   then  unzip   
This directory will have sub-directories like lib, include, data, and so on.   
```
ls TensorRT-${version}   

bin  data  doc  include  lib  onnx_graphsurgeon  python  samples  targets
```
 
https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html#installing-tar

**Comment by tadejsv** [comment]
Thanks, I don't want to download and unzip the file manually, I want to do it using `pip` so that I can properly specify my environment and resolve dependencies. Please fix the issue with missing versions in NVIDIA's python package index

**Comment by zerollzeng** [comment]
Checking internally.

**Comment by zerollzeng** [comment]
> `onnx-graphsurgeon` only works with python versions up to 3.8.

it should be python version agnostic, could you please provide a reproduce of the issue? 
thanks!

**Comment by tadejsv** [comment]
> > `onnx-graphsurgeon` only works with python versions up to 3.8.
> 
> it should be python version agnostic, could you please provide a reproduce of the issue? thanks!

Yeah you're right - this was actually a problem with newer versions of onnx-graphsurgeon being available. However, you can still change.

However, it would be good to update classifiers in `setup.py` to the python versions actually supported:

https://github.com/NVIDIA/TensorRT/blob/28733f0fdccde2967fed395b06ca491af3a561a9/tools/onnx-graphsurgeon/setup.py#L48-L55

**Comment by zerollzeng** [comment]
@pranavm-nvidia ^ ^
